# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/ViewportFrame.cpp
# Commit: c428c3ad7725
# Full Hash: c428c3ad7725593f33b4ea2da9851be31be0c1df
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-11-28 15:41:50
# Regressor Bug: 1791759
# File Overlap Count: 1
# Description:
#   Bug 1791759 - Add support for `content-visibility: auto` r=emilio
#   
#   This change adds support for `content-visibilty: auto` as well as
#   showing and hiding content based on the relevancy of the content as
#   defined in the specification. Changes to relevancy are handled by
# ==============================================================================

diff -r ce553434720c -r c428c3ad7725 layout/generic/ViewportFrame.cpp
--- a/layout/generic/ViewportFrame.cpp	Mon Nov 28 09:45:39 2022 +0000
+++ b/layout/generic/ViewportFrame.cpp	Mon Nov 28 09:53:06 2022 +0000
@@ -177,7 +177,8 @@
       continue;
     }
 
-    if (frame->IsHiddenByContentVisibilityOnAnyAncestor()) {
+    if (frame->IsHiddenByContentVisibilityOnAnyAncestor(
+            nsIFrame::IncludeContentVisibility::Hidden)) {
       continue;
     }
 